bootstrap.sh

```diff
@@ -569,20 +569,38 @@ install_salt_formula_git()
     [ ! -L /srv/salt/env/dev ] && ln -s /usr/share/salt-formulas/env /srv/salt/env/dev || echo ""
 }
 
+saltservice_stop() {
+    $SUDO service salt-minion stop
+    $SUDO service salt-master stop
+    sleep ${SALT_STOPSTART_WAIT:-30}
+    pgrep salt-master | sed /$$/d | xargs --no-run-if-empty -i{} $SUDO kill -9 {} && rm -rf /var/run/salt/master/* || true
```

Comment:
pkill -9 -f salt-master && rm -rf /var/run/salt/master/* || true

Reply:
We can't used it before because our scripts was called salt-master-bootstrap.sh which caused to stop the script itself instead of salt master.
pkill /usr/bin/salt-master

Reply:
ok, pkill didn't worked at time when the bootstrap.sh was named "salt-master-init.sh"

Reply:
`pkill -9 -f /usr/bin/salt-master && rm -rf /var/run/salt/master/* || true` tested and working.